### ISSUE TITLE
[BUG] Banner ads are present in the scene, clickable, but are invisible.

### ISSUE THREAD

**Issue by ajk2196** [Poing-Studios/godot-admob-android#137]
**Describe the bug**
I am trying to show my banner ads in my app. When I call them using the show_banner() call, nothing seems to happen, but in reality, the ads do load and get in the place where they are supposed to be, but for some reasons they remain invisible. Upon tapping on the area where they are supposed to be, does open something related to a test ad - which means that it is there, present on the screen, just not showing up.

**To Reproduce**
I am right now using this to show the banner ads:

        MobileAds.load_banner()
	MobileAds.show_banner()

**Expected behavior**
The banner ads should be visible, not invisible/transparent.

**Screenshots**
I have attached two videos that show the problem.

**Smartphone (please complete the following information):**
 - Device: Redmi POCO Phone F1
 - OS: MIUI Global
 - Version 12.0.3

I do not think that this problem is device/OS specific. I have tested my app on my brother's and parents' phones. All are different models and exhibit same behaviour.

**Additional context**
I have attached two videos here:

1. First video has my app in Aspect > KeepWidth mode. It has invisible ads.
2. Second video has my app is Aspect > KeepHeight mode. It has only partially visible ads at the bottom of the screen.

https://user-images.githubusercontent.com/90527174/189789607-d7412723-7f0f-4915-97af-e16aeaacc697.mp4

https://user-images.githubusercontent.com/90527174/189790058-87184df0-3c56-4f29-9410-b260dc30ff92.mp4

**Comment by gumaciel** [comment]
Hi! What Godot version and Android Plugin version are you using?

**Comment by ajk2196** [comment]
Godot V3.4.4 stable and for Admob version, I have attached the screenshot, I think it is 1.2.4 or 2.0.0 for Android.
![Screenshot (168)](https://user-images.githubusercontent.com/90527174/190943539-414969a3-c031-4b57-b6c2-8650be77b670.png)

**Comment by gumaciel** [comment]
@ajk2196 Can you provide me a sample project of this bug? I can't reproduce here

I find it strange that it seems that your UI is in front of the ads, are you using any other Plugin for Android besides our AdMob?

**Comment by ajk2196** [comment]
It is hard to reproduce this, I have tried this combination of plugin version and Godot version in some of my projects. Everything is normal in my other projects - the ads appear normally. Only this particular project is having this weird issue. I even tried loading it with latest plugin version and Godot version 3.5, but the bug still remained. It is hard to determine what is causing this bug. I will shortly be redoing this same entire project from scratch in about a month when I have time - everything step by step to pinpoint the change that triggers this bug. I will post about it when done. Hopefully, that shall help in solving this bug.

The interstitial ads show normally. Only the banner ads are getting overlayed by the app's scenes.

**Comment by gumaciel** [comment]
Thank you @ajk2196, if you have any news, please let me know

**Comment by patrick-wagner-0681** [comment]
Do you have Per Pixel Transparency enabled ?

**Comment by ajk2196** [comment]
Yes. I don't have access to my project right now as I am on my phone, but I may have it on, I have it on by default in all my projects, so there are high chances that it may be on in this project as well. I will try toggling it and post the results once I am home. Thanks for the hint @patrick-wagner-0681

**Comment by ajk2196** [comment]
I have finally built my project from scratch and thankfully, have found no issues. Like @patrick-wagner-0681 suggested, I did not turn on Per Pixel Transparency, but also, I did not dare to mess with it with the fear of the problem getting repeated again. It took me almost a month to re-assemble my project from scratch.

Things I noted from this bug:

1. The bug was project specific. Making trial Android apps to test ads displayed the Ads just fine.

2. Removing / uninstalling Godot, Admob plugins, Android Build Templates and then adding / installing their same/different versions again did not help. The only thing that helped was re-making and a little bit of re-assembling the app from scratch.

3. To be sure that everything goes well, I 'Refreshed' my PC. It runs on Windows 10, where Refreshing a PC means keeping personal files and removing all installed softwares on the PC.
